frame/nomination-pools/src/mock.rs

```diff
@@ -123,6 +111,48 @@ impl sp_staking::StakingInterface for StakingMock {
 	fn nominations(_: Self::AccountId) -> Option<Vec<Self::AccountId>> {
 		Nominations::get()
 	}
+
+	fn stash_by_ctrl(_controller: &Self::AccountId) -> Result<Self::AccountId, DispatchError> {
+		todo!()
+	}
+
+	fn stake(who: &Self::AccountId) -> Result<Stake<Self>, DispatchError> {
+		match (
+			UnbondingBalanceMap::get().get(who).map(|v| *v),
+			BondedBalanceMap::get().get(who).map(|v| *v),
+		) {
+			(None, None) => Err(DispatchError::Other("balance not found")),
+			(Some(v), None) => Ok(Stake { total: v, active: 0, stash: who.clone() }),
+			(None, Some(v)) => Ok(Stake { total: v, active: v, stash: who.clone() }),
+			(Some(a), Some(b)) => Ok(Stake { total: a + b, active: b, stash: who.clone() }),
+		}
+	}
+
+	fn election_ongoing() -> bool {
+		todo!()
```

Comment:
```suggestion
		unimplemented!("reason?")
```

Reply:
Yep, basically we don't need those methods for nom-pools tests, I will make sure this is properly documented via reason.